content/en/developers/integrations/agent_integration.md

```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
+For installing your wheel to test in containerized environments: 
+1. Mount the whl into a initContainer
+2. Run the whl install in the init container
+3. Mount the init container in the agent container while it's running
```

Comment:
We're gonna need some commands for these.

Reply:
I think this is something they configure, not something we instruct them to already have. Agent would be best to loop in for commands if we need them @hestonhoffman